src/Drive.php

```diff
@@ -380,7 +409,7 @@ public function uploadFileStream(string $path, $resource): bool
     }
 
     /**
-     * delete resource
+     * Delete (move to trash-bin) the Ocis resource (file or folder) at the given path in the current drive
```

Comment:
Does every ocis have a trash bin?
Or is it possible to configure ocis without a trash bin?

Reply:
I think every storage driver is supposed to implement a trash
`After deleting a node the storage allows listing the deleted nodes and has an undo mechanism for them.` from:
https://owncloud.dev/ocis/storage/storagedrivers/#trash-persistence